### ISSUE TITLE
Update all phone number links using new aria-label source code

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#10003]
## Status
**[2024-07-12] [Fran]** Question to Laura in [Slack ](https://agilesix.slack.com/archives/C05Q7J61NR5/p1720817218727949)as to whether this old ticket is still needed.

## User story
As a Veteran using a screen reader, I need to be able to understand a phone number when my screen reader speaks it.

## Issue Description
Currently all phone numbers in Drupal pages are hyperlinked using the link button. This is a 508 accessibility issue because screen readers can often read numbers as hundreds or thousands. Adding an aria label to the source code allows the screen reader to read each number separately with a slight pause like you would say a phone number. 

A content editor needs to update all instances of hyperlinked phone numbers with the following source code. In Drupal this means not using the link button and instead adding it while in source mode.

`<a href="tel:+18008271000" aria-label="8 0 0. 8 2 7. 1 0 0 0.">800-827-1000</a>`
 
---
## Tasks
Update all phone number instances in each hub using source code listed above. (Content Writer - TBD)
- [ ] Health care
- [ ] Disability
- [ ] Education
- [ ] Careers and employment
- [ ] Pension
- [ ] Housing assistance
- [ ] Burials and memorials
- [ ] Life insurance
- [ ] Records
- [ ] Service member benefits
- [ ] Family member benefits

Updates reviewed, published (Content editor - Randi)

## Acceptance Criteria
- [ ] All phone numbers on live site contain an aria label 
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by FranECross** [comment]
@FranECross to ask Laura about this in their weekly meeting.

**Comment by FranECross** [comment]
@laflannery Is this still needed? Thanks!
